src/Features/Lsif/Generator/Generator.cs

```diff
@@ -324,15 +347,30 @@ void MarkImplementationOfSymbol(ISymbol baseMember)
             }
 
             lsifJsonWriter.Write(Edge.Create("contains", documentVertex.GetId(), rangeVertices, idFactory));
```

Comment:
Move this writing of this contains edge back to GenerateForDocumentAsync since that's where the document object was originally created. That way methods creating vertices are responsible for the connection of them.

Reply:
sure :)

Reply:
done.